crates/starknet-devnet/tests/benches/mint_bench.rs

```diff
@@ -0,0 +1,36 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion, SamplingMode};
+use tokio::runtime::Runtime;
+
+use crate::common::background_devnet::BackgroundDevnet;
+
+#[path = "../common/mod.rs"]
+pub mod common;
+
+static DUMMY_ADDRESS: u128 = 1;
+static DUMMY_AMOUNT: u128 = 1;
+
+async fn mint_iter(f: &str) {
+    let devnet =
+        BackgroundDevnet::spawn_with_additional_args(&[
+            format!("--state-archive-capacity={}", f).as_str()
+        ])
+        .await
+        .expect("Could not start Devnet");
+
+    for _n in 1..5000 {
+        devnet.mint(DUMMY_ADDRESS, DUMMY_AMOUNT).await;
+    }
+}
+
+fn bench_memory(c: &mut Criterion) {
+    let rt = Runtime::new().unwrap();
+    let mut group = c.benchmark_group("Mint");
+    group.significance_level(0.1).sample_size(10);
```

Comment:
I would limit the sample size to 5, with state-archive-capacity full 10 samples take ages even on high end machines

Reply:
never mind I can see that 10 is min in criterion